CHANGELOG.md

```diff
@@ -22,8 +22,7 @@ and this project adheres to
 - `cryptoPrefetcher()` to throw when `bufSize` is not a multiple of
   `valueSize`.
 - `MemoryWritable` internal buffer size is now limited to 8 MB by default.
-- Signature of `callerFilepath()` to allow passing `RegExp` as depth to be used
```

Comment:
Why did you delete the existing changelog entry? Revert this change, please.

Reply:
oh, sorry. This happened during rebase. Didn't noticed it